google/cloud/storage/bucket_metadata.h

```diff
@@ -142,11 +142,11 @@ std::ostream& operator<<(std::ostream& os, CorsEntry const& rhs);
  *
  * @see Before enabling Uniform Bucket Level Access please
  *     review the [feature documentation][bpo-link], as well as
- *     ["Should you use Bucket Policy Only?"][bpo-should-link].
+ *     ["Should you use Uniform bucket-level access ?"][bpo-should-link].
  *
- * [bpo-link]: https://cloud.google.com/storage/docs/bucket-policy-only
+ * [bpo-link]: https://cloud.google.com/storage/docs/uniform-bucket-level-access
```

Comment:
`bpo-link` can be changed to `ubla-link`. 

Please check name on other places of this PR.